back-office/catalog/products/catalog-products-add-edit.md

```diff
@@ -55,7 +56,7 @@ Keyboard shortcut: ALT + SHIFT + V
 * **Save**: Button to save all changes made in all tabs.      
 Keyboard shortcut: ALT + SHIFT + S.      
 If it’s saved with success, there is a green success message “Settings updated”. Otherwise, there is an error message.
-* **Duplicate**: When you click on this button, it saves and duplicate the current product \(the duplicated product will be offline\). The duplicated product must be exactly the same. 
+* **Duplicate**: When you click on this button, it saves and duplicate the current product \(the duplicated product will be offline\). The duplicated product has all the product information (product associated, carriers configuration, associated suppliers, infromation of all the combinaition and the associated files) of the copied product.
```

Comment:
```suggestion
* **Duplicate**: When you click on this button, it saves and duplicates the current product \(the duplicated product will be offline\). The duplicated product has all the product information (related products, shipping configuration, associated suppliers, combinations, and attached files) of the copied product.
```

Reply:
I've just updated the list with the same terminology used in the product page to make it clearer. ;-)